modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -130,6 +130,14 @@ using GRunArgP = util::variant<
     >;
 using GRunArgsP = std::vector<GRunArgP>;
 
+inline std::vector<GRunArgP>& operator += (std::vector<GRunArgP> &lhs, const std::vector<GRunArgP> &rhs);
```

Comment:
Maybe you can also implement operator+() to provide opportunity to add multiple Runargs/Protoargs at the same time in one line. If it's not going to take too much effort, of course

Reply:
Good point

Implementation of operator+() for GRunArgsP is simple
But implementation of operator+() for GIOProtoArgs is not so obvious
So i left the corresponding TODO comment